ups/obs_subaru.cfg

```diff
@@ -11,5 +11,5 @@ dependencies = {
 config = lsst.sconsUtils.Configuration(
     __file__,
     hasDoxygenInclude=False,
-    hasSwigFiles=True,
+    hasSwigFiles=False,
```

Comment:
I suggest deleting this line (despite the fact that the current default is True)